isis/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 
 # Specify the required version of CMake.  If your machine does not
 #  have this, it should be easy to build from https://cmake.org/download/
-cmake_minimum_required(VERSION 3.10)
+#cmake_minimum_required(VERSION 3.10)
```

Comment:
Why did this get axed? we need 3.10 for ctest/gtest integration

Reply:
Oh no, this was unintentional. Thanks for catching it.